editor/inspector/assets/fbx/animation.js

```diff
@@ -683,6 +685,15 @@ exports.methods = {
     },
 
     onSelect(rawClipIndex, splitClipIndex) {
+        const animInfo = this.animationInfos[this.rawClipIndex];
+
+        if (!animInfo) {
+            rawClipIndex = 0;
+        }
+        const splitInfo = animInfo.splits[this.splitClipIndex];
```

Comment:
if above code `!animInfo` is true, here would occur error.
`onSelect` method，these changes are unnecessary